server/src/test/java/io/crate/integrationtests/PostgresITest.java

```diff
@@ -1100,6 +1100,30 @@ public void test_metadata_calls_do_not_query_pg_catalog_tables_repeatedly() thro
         }
     }
 
+    @Test
+    public void test_table_function_without_from_can_bind_parameters() throws Exception {
```

Comment:
Not sure it's needed, created a planner test as well - should I drop it?

Reply:
I think planner test would be good enough, but also fine to keep it. No strong opinion

Reply:
will keep it for now